src/cmd-fetch

```diff
@@ -5,6 +5,18 @@ dn=$(dirname "$0")
 # shellcheck source=src/cmdlib.sh
 . "${dn}"/cmdlib.sh
 
+FILE=cache/pkgcache-repo
+if [ -d "${FILE}" ]
+then
+	pkgcachesize=$(du --bytes --max-depth 0 "${FILE}" \
+                       | awk '{print $1; exit}')
+        pkglimit=$((1024 * 1024 * 1024 * 5))
```

Comment:
whitespace? The indentation looks a bit off.

Reply:
Thanks. Fixed it.